commit 70c7273778bf7f18f2e46a41638f6ff38fb9fa51
Author: Priyanka Dandamudi <priyanka.dandamudi@intel.com>
Date:   Wed Feb 12 09:32:12 2025 +0000

    drm/xe: Add fault injection for xe_sync_entry_parse
    
    Add fault injection for xe_sync_entry_parse to allow it to fail while
    executing xe_vm_bind_ioctl(). This needs to be added as it cannot be
    reached by injecting error through IOCTL arguments.
    
    Signed-off-by: Priyanka Dandamudi <priyanka.dandamudi@intel.com>
    Reviewed-by: Satyanarayana K V P <satyanarayana.k.v.p@intel.com>
    Reviewed-by: Himal Prasad Ghimiray <himal.prasad.ghimiray@intel.com>
    Link: https://patchwork.freedesktop.org/patch/msgid/20250212093212.3069356-1-priyanka.dandamudi@intel.com
    Signed-off-by: Himal Prasad Ghimiray <himal.prasad.ghimiray@intel.com>

diff --git a/drivers/gpu/drm/xe/xe_sync.c b/drivers/gpu/drm/xe/xe_sync.c
index 42f5bebd09e5..f87276df18f2 100644
--- a/drivers/gpu/drm/xe/xe_sync.c
+++ b/drivers/gpu/drm/xe/xe_sync.c
@@ -210,6 +210,7 @@ int xe_sync_entry_parse(struct xe_device *xe, struct xe_file *xef,
 
 	return 0;
 }
+ALLOW_ERROR_INJECTION(xe_sync_entry_parse, ERRNO);
 
 int xe_sync_entry_add_deps(struct xe_sync_entry *sync, struct xe_sched_job *job)
 {